index.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+// ES6 is always in strict mode
```

Comment:
This is not correct. It's always strict inside arrow functions, class methods, and modules. The top-level in Node.js is not yet a ES2015 module, so it's not strict by default.

Reply:
You are already using arrow functions and an exported module in this project.